docs/config-file-ort-yml.md

```diff
@@ -27,19 +27,34 @@ Exclusions apply to paths (files/directories) or scopes. Examples of currently s
 
 ### Excludes Basics
 
-ORT's philosophy is to analyze and scan everything it can find to build a complete picture of a repository and its
-dependencies.
+ORT's default philosophy is to analyze and scan everything it can find to build a complete picture of a repository and
+its dependencies.
 
 However, the users may not be interested in the results for components that are not included in their released
 artifacts, for example build files, documentation, examples or test code. To support such use cases, ORT provides a
 mechanism to mark files, directories or scopes included in the repository as excluded.
 
-Note that the excluded parts are analyzed and scanned, but are treated differently in the reports ORT generates:
+Note that in the standard mode the excluded parts are analyzed and scanned, but are treated differently in the reports
+ORT generates:
 
 * The issue summary does not show issues in the excluded parts.
 * The excluded parts are grayed out.
 * The reason for the exclusion is shown next to the result.
 
+This is a rather safe option, since the reports still display elements marked as excluded and thus allow the user to
```

Comment:
"allow*s*"

Reply:
Not sure about this one: The "allow" refers to "reports" as subject.

Reply:
You're right.